apps/masterbots.ai/lib/hooks/use-thread.tsx

```
@@ -113,85 +118,85 @@ export function ThreadProvider({ children }: ThreadProviderProps) {
-      setIsOpenPopup(true)
+      const fullMessage = bulletContent;
+
+      setState({ isNewResponse: true, isOpenPopup: true });
       await saveNewMessage({
-        role: 'user',
+        role: "user",
         threadId: activeThread?.threadId,
         content: fullMessage,
-        jwt: session!.user?.hasuraJwt
-      })
+        jwt: session!.user?.hasuraJwt,
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)